src/OpenTelemetry.Instrumentation.AspNet/Implementation/HttpInListener.cs

```diff
@@ -179,7 +179,10 @@ public override void OnStopActivity(Activity activity, object payload)
                 activityToEnrich.SetTag(SemanticConventions.AttributeHttpStatusCode, response.StatusCode);
 
                 Status status = SpanHelper.ResolveSpanStatusForHttpStatusCode(response.StatusCode);
```

Comment:
`ResolveSpanStatusForHttpStatusCode` can be moved inside the if check

Reply:
fixed! thanks for the code review! :)